docs/atmosphere/index.rst

```diff
@@ -0,0 +1,47 @@
+.. _atmosphere:
+
+==========================
+ Atmosphere (`atmosphere`)
```

Comment:
This PR seems to be about nectarcam event source ... but some things related to the atmosphere seem to have somehow found their way into this PR... 

Maybe it is better to remove this for the moment from this PR and open a dedicated "Atmosphere" PR

Reply:
You are right, strange, it should have been pulled already some time ago. How can take it back from this PR?

Reply:
there are multiple ways:

### simple
just `git rm` these 4 `__init__.py` files and the `index.rst` file that were accidentally added.

git tries to make sure we never loose, when ever we have written once, so this "adding-and-removing" will be part of the history of this branch and also later part of the history of the master. Maybe not ideal, but imho also no big deal.

### cherry-pick

Make a fresh branch from the master (after making sure your fork is up-to-date) and give it a slightly different name and `git cherry-pick` only those commits from this branch, which you actually want to have in a commit.

### ... more

There are more ways ... I have never tried....

Reply:
opened a little PR for this here:
https://github.com/FrancaCassol/ctapipe/pull/2